src/main/java/uk/gov/hmcts/reform/cpo/repository/CasePaymentOrdersRepositoryImpl.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.cpo.repository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.cpo.data.CasePaymentOrderEntity;
+import uk.gov.hmcts.reform.cpo.exception.CasePaymentOrderCouldNotBeFoundException;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public class CasePaymentOrdersRepositoryImpl implements CasePaymentOrdersRepository {
+    private final CasePaymentOrdersJpaRepository casePaymentOrdersJpaRepository;
+
+    private final CasePaymentOrdersAuditJpaRepository casePaymentOrdersAuditJpaRepository;
+
+    private static final String ERROR_MESSAGE = "One of the supplied %s's was not present in the database";
```

Comment:
Feel free to add to ValidationError messages if you think it fits: I am unsure.

Reply:
Resolving as code no longer applicable after recent refactored of exception handling.